apps/emqx_resource/src/emqx_resource_buffer_worker.erl

```diff
@@ -1122,30 +1122,28 @@ pre_query_channel_check({Id, _} = _Request, Channels, QueryOpts) when
         true ->
             ok;
         false ->
-            maybe_throw_channel_not_installed(Id, QueryOpts)
+            error_if_channel_is_installed(Id, QueryOpts)
     end;
 pre_query_channel_check({Id, _} = _Request, _Channels, QueryOpts) ->
-    maybe_throw_channel_not_installed(Id, QueryOpts);
+    error_if_channel_is_installed(Id, QueryOpts);
 pre_query_channel_check(_Request, _Channels, _QueryOpts) ->
     ok.
 
-maybe_throw_channel_not_installed(Id, QueryOpts) ->
+error_if_channel_is_installed(Id, QueryOpts) ->
     %% Fail with a recoverable error if the channel is not installed and there are buffer
```

Comment:
the function is named if...is_installed,
however the comment is about error if channel is "not" installed.

Reply:
Thanks, fixed.